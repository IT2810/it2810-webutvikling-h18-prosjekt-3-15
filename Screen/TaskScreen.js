import React from 'react';
import {View} from 'react-native';
import Todo from '../components/Todo';
class TaskScreen extends React.Component {
    //TaskScreen renders the To-do component, and sends down the navigation prop from App.js.
    render() {
        return (
            <View>
                <Todo navigation={this.props.navigation}/>
            </View>
        );
    }
}
export default TaskScreen;