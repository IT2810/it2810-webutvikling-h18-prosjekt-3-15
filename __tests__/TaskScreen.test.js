import React from 'react';
import renderer from 'react-test-renderer';
import TaskScreen from "../Screen/TaskScreen";
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<TaskScreen navigation={{'state':{'params':{'url': '/1/test/'}}}}/>);
    expect(tree).toMatchSnapshot();
});

