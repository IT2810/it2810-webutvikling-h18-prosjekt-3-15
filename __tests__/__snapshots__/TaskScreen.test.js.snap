// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component>
  <Todo
    navigation={
      Object {
        "state": Object {
          "params": Object {
            "url": "/1/test/",
          },
        },
      }
    }
  />
</Component>
`;
